website/src/lib/services/survey/survey.service.ts

```diff
@@ -426,4 +427,97 @@ export class SurveyService extends BaseService {
 			return this.resultFail('Could not fetch surveys');
 		}
 	}
+
+	async getByEmail(email: string): Promise<ServiceResult<SurveyPayload>> {
```

Comment:
could we rename to getByAccessEmail to make more clear its this "pseudo" email and not the real email of the recipient